src/app/screens/App/components/PrivateRoute/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CustomRoute from '../CustomRoute';
+
+function isAllowedAccess() {
```

Comment:
i'd change this function to `hasUserAccess` and i'd move to a `utils` folder into `storageUtils` file.
So you avoid repeat the code `(-- REF1--)`